libcontainer/state_linux.go

```diff
@@ -64,7 +64,7 @@ func runPoststopHooks(c *Container) error {
 	}
 	s.Status = specs.StateStopped
 
-	if err := hooks[configs.Poststop].RunHooks(s); err != nil {
+	if err := hooks.Run(configs.Poststop, s); err != nil {
```

Comment:
So, after update `golangci-lint`'s version, it's more clever than before.
I hope I can have such good eyes. 👁️

Reply:
Fixed now

Reply:
@lifubang ptal